schema/defs.yaml

```diff
@@ -313,3 +314,21 @@ description: Common schema definitions shared by all themes
         moving along the line from beginning to end
       properties:
         side: { "$ref": "#/$defs/propertyDefinitions/side" }
+    cartographyContainer:
+      title: cartography
+      description: Defines cartograhic hints for optimal use of Overture features in map making.
+      type: object
+      properties:
+        cartography:
+          type: object
+          properties:
+            min_zoom:
+              description: Recommended minimum tile zoom per the [Google Maps](https://www.maptiler.com/google-maps-coordinates-tile-bounds-projection/#3/15.00/50.00) tiling specification.
```

Comment:
Since markdown links are not working in documentation, could you delete [Google Maps] part and just leave blank URL?